app.js

```diff
@@ -0,0 +1,82 @@
+'use strict';
+var express = require('express');
+var trucks = require('./trucks');
+var _ = require('underscore')
+var bodyParser = require('body-parser')
+var app = express();
+app.use(bodyParser.urlencoded({extended: true}));
+app.use(bodyParser.json());
+app.use(express.static('public'));
+
+app.get('/trucks', function (request, response) {
+  response.json(trucks.getTrucks());
+});
+
+app.post('/trucks', function (request, response){
+  var newTruck = request.body;
+  var foodTrucks = trucks.getTruck();
+  console.log(newTruck);
+  if (newTruck) {
+    var truckNames = _.pluck(foodTrucks, 'name')
+    var contains = _.contains(truckNames, newTruck.name)
+    if (!contains) {
+      trucks.addTruck(newTruck)
+      response.status(201).json('Truck added');
+    } else {
+      response.status(400).json('Truck already exists');
+    }
+  }else {
+    response.status(400).json('Truck not added');
+  }
+});
+
+app.get('/trucks/:name', function (request, response) {
+  var truck = trucks.getTruck(request.params.name);
+  if (truck) {
+    response.json(trucks.getTruck(request.params.name));
```

Comment:
You've already assigned the result of the call to the variable 'truck' so just send it to the response.json() call.